scripts/build.sh

```diff
@@ -21,12 +21,10 @@
 dir=$(dirname $0)
 projectRoot=${dir}/..
 
-if [ -z "$DOCKER_NAMESPACE" ]; then
-  set -- $@ "-Ddocker.project.namespace=$DOCKER_NAMESPACE"
-fi
+source ${projectRoot}/scripts/utils/maven.sh
 
-pushd $projectRoot
-mvn -P-local-docker-build -P-test.local "$@" clean install
+pushd ${projectRoot}
+  maven_utils::execute $@ clean install
```

Comment:
Indeed, `maven_utils::execute` can be used to pass arbitrary arguments so this is not needed anymore